newton/_src/sim/model.py

```
@@ -813,6 +816,22 @@ def control(self, requires_grad: bool | None = None, clone_variables: bool = Tru
             c.tri_activations = self.tri_activations
             c.tet_activations = self.tet_activations
             c.muscle_activations = self.muscle_activations
+
+        # Initialize kinematic_target from joint_q, zeroing velocity-mode entries
+        if self.joint_count and self.joint_kinematic_mode is not None:
+            from .joints import KinematicMode  # noqa: PLC0415
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)